presto-docs/src/main/sphinx/admin/resource-groups.rst

```diff
@@ -85,6 +85,9 @@ Selector Properties
     * ``INSERT``: ``INSERT`` and ``CREATE TABLE AS SELECT`` queries.
     * ``SELECT``: ``SELECT`` queries.
 
+* ``resourceEstimatesRange`` (optional, experimental): resource estimates range match against the resource estimates
```

Comment:
How about `resourceEstimatesRange` -> `resourceEstimate` or `resourceUsageEstimate` ?

And then in the description we can clarify that it's a range. We don't need to put the "range" in the name of this property. We can say something like: `a range that specifies a resource usage estimate to match against the resource usage estimate of a query`. /cc @electrum 

Please also update the example JSON in this file (below) to show how this property can be set.